templates/docs/examples/patterns/contextual-menu/dark.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
 <span class="p-contextual-menu--left is-dark">
-    <button class="p-contextual-menu__toggle" aria-controls="menu-3" aria-expanded="false" aria-haspopup="true">Take action</button>
+    <button class="p-contextual-menu__toggle" aria-controls="menu-3" aria-expanded="false" aria-haspopup="true">Take action...</button>
```

Comment:
Make it an ellipsis, not 3 full stops (in all examples plz):

```suggestion
    <button class="p-contextual-menu__toggle" aria-controls="menu-3" aria-expanded="false" aria-haspopup="true">Take action…</button>
```

Reply:
Ah yes - done